src/api/layer/linked_accounts.ts

```diff
@@ -29,11 +36,12 @@ export const denyConnection = post<
     accountId: string
   }
 >(
-  ({businessId, accountId}) => `/v1/businesses/${businessId}/external-accounts/${accountId}/confirm`
+  ({ businessId, accountId }) =>
+    `/v1/businesses/${businessId}/external-accounts/${accountId}/deny`,
 )
 
 // TODO: not implemented yet in backend
```

Comment:
We can drop this comment now, right?

Reply:
This remains accurate. Right now we are directly using `unlinkPlaidItem`